### ISSUE TITLE
Absent hippocampal commissure

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6713]
**Preferred term label:**
Absent hippocampal commissure

**Synonyms**
Agenesis...

**Definition (free text, please give PubMed ID)**
Absence of the fibers that connect the contralateral hippocampi via the crura of the fornix and run beneath the posterior portion of the corpus callosum.

**Parent term (use hpo.jax.org/app)**
Abnormal morphology of the hippocampus HP:0025100

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 617542 , Gaze palsy, familial horizontal, with progressive scoliosis, 2 
3/3  PMID:28250456

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
